GSOC.md

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
Do we need this file in the main repo @theosotr?

Reply:
I forgot it in there. Sorry! Removing it now.